tests/Model/LazyLoading/AdvancedManyToManyObjectRelationTest.php

```diff
@@ -83,6 +84,24 @@ public function testClassAttributes()
         }
     }
 
+    public function testDirty()
+    {
+        $object = $this->createDataObject();
+
+        $relatedObjects = $this->loadMetadataRelations('advancedObjects', 'metadataUpper');
+
+        $object->setAdvancedObjects($relatedObjects);
+        $object->save();
+        $this->assertFalse($object->isDirty('advancedObjects'), 'Advanced relation must not be dirty after saving');
```

Comment:
maybe isFieldDirty instead ?

Reply:
Thanks, fixed this.
Do you know where the remaining problems come from, @weisswurstkanone ?
```
 ------ ---------------------------------------------------------------- 
  Line   bundles/AdminBundle/Controller/Admin/TranslationController.php  
 ------ ---------------------------------------------------------------- 
  1085   Left side of && is always true.                                 
 ------ ---------------------------------------------------------------- 

 ------ --------------------------------------------------------------------- 
  Line   bundles/AdminBundle/Controller/Searchadmin/SearchController.php      
 ------ --------------------------------------------------------------------- 
  525    Comparison operation ">=" between int<0, max> and 0 is always true.  
 ------ --------------------------------------------------------------------- 

Error: ] Found 2 errors                                                         

Error: Left side of && is always true.
Error: Comparison operation ">=" between int<0, max> and 0 is always true.
Error: Process completed with exit code 1.
```

I tried to merge the base branch but with no success.

Reply:
> Thanks, fixed this.
> Do you know where the remaining problems come from, @weisswurstkanone ?
> 
> ```
>  ------ ---------------------------------------------------------------- 
>   Line   bundles/AdminBundle/Controller/Admin/TranslationController.php  
>  ------ ---------------------------------------------------------------- 
>   1085   Left side of && is always true.                                 
>  ------ ---------------------------------------------------------------- 
> 
>  ------ --------------------------------------------------------------------- 
>   Line   bundles/AdminBundle/Controller/Searchadmin/SearchController.php      
>  ------ --------------------------------------------------------------------- 
>   525    Comparison operation ">=" between int<0, max> and 0 is always true.  
>  ------ --------------------------------------------------------------------- 
> 
> Error: ] Found 2 errors                                                         
> 
> Error: Left side of && is always true.
> Error: Comparison operation ">=" between int<0, max> and 0 is always true.
> Error: Process completed with exit code 1.
> ```
> 
> I tried to merge the base branch but with no success.

These issues are fixed on base branch 10.1. please rebase your PR on 10.1. thanks!

Reply:
Thanks @dvesh3 - I rebased 10.1 and now tests are successful.